main/tests/cypress/cypress/integration/project/grid/misc/proper-display.js

```diff
@@ -0,0 +1,23 @@
+describe(__filename, function () {
+  it('proper tag assingnment to each text snippet', function () {
+    const fixture = [
+      ['tests'],
+      ['2021-01-31https://www.google.com'],
+      [
+        'https://www.wikidata.org/wiki/Property:P670 https://www.wikidata.org/wiki/Property:P669 are now mapped to https://schema.org/streetAddress via https://www.wikidata.org/wiki/Property:P2235',
+      ],
+      ['vhjhjjj https://github.com/OpenRefine/OpenRefine/issues/2519'],
+    ];
+    cy.loadAndVisitProject(fixture);
+
+    cy.getCell(0, 'tests').contains('2021-01-31https://www.google.com');
+    cy.getCell(1, 'tests')
```

Comment:
Can this test also check if there is `span` too as this is only checking the presence of `href`?

Reply:
yes